rootfs/standard/var/www/mynode/templates/includes/head.html

```diff
@@ -0,0 +1,15 @@
+<link rel="shortcut icon" href="{{ url_for('static', filename='images/favicon.ico') }}">
+<link href="{{ url_for('static', filename='jquery_ui/jquery-ui.css')}}" rel="stylesheet">
+{% if ui_settings['darkmode'] %}
```

Comment:
To prevent code duplication, would it be possible to make the dark mode a separate CSS file that gets loaded on top of the existing CSS file?

Like

include normal css
if darkmode:
    include dark mode css

Where the dark mode css only has the changes to make the UI dark.

Reply:
that will be ideal, I'll try it out and push it in few days

Reply:
Done!